### PR TITLE
Some modifications

### PR DESCRIPTION
**Pull request by glixx**

### CODE REVIEW

File: src/flam3util.h [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 #undef VERSION
 extern "C" {
-#include "flam3.h"
+#include "flam3/flam3.h"
```
Comment (bitsed) [inline src/flam3util.h]: The INCLUDEPATH in qosmic.pro or your pkg-config can adjusted to locate flam3.h on your system.
I think this change should be removed from your request.

File: src/qosmic.cpp [hunk 2]
```diff
@@ -39,6 +40,26 @@ int main(int argc, char* argv[])
 	Logger::getInstance()->setLevel(Logger::levelFor(getenv("log")));
 	logInfo(QString("main() : Qosmic (version %1)").arg(QOSMIC_VERSION));
 
+    // Translations for Qt strings
```
Comment (bitsed) [inline src/qosmic.cpp]: The translations are loaded again in the following block.  I don't see the problem solved by this change.
I think this change should be removed from your request.

**Comment by glixx** [comment]
You use translations for qosmic_ from resources, but it's not enough. This patch uses translations for qt_ from system too. Otherwise all qt_ strings are untranslated: context menu, buttons, etc.

**Comment by bitsed** [comment]
I think you are correct.  Without this patch any translations for Qt framework dialogs are not found.
I'm going to merge your changes.  Thank you again.

**Pull request merged**